analyses/molecular-subtyping-pathology/results/compiled_molecular_subtypes_with_clinical_pathology_feedback_and_report_info.tsv

```diff
@@ -4669,8 +4669,8 @@ PT_FC0EF5ZT	7316-3572	BS_WJ7GMQTF	Low-grade astrocytic tumor	Ganglioglioma	Gangl
 PT_FCM7XPVR	7316-2285	BS_4GKH983E	Ependymal tumor	Ependymoma	NA	EPN, To be classified	NA	Recurrence	NA	NA
 PT_FCM7XPVR	7316-2285	BS_554AS76R	Ependymal tumor	Ependymoma	NA	EPN, To be classified	NA	Recurrence	NA	NA
 PT_FCM7XPVR	7316-2285	BS_MQCKXD60	Ependymal tumor	Ependymoma	NA	EPN, To be classified	NA	Recurrence	NA	NA
-PT_FCWMT41S	7316-7133	BS_4Q7PQHS8	Diffuse astrocytic and oligodendroglial tumor	HGAT	NA	DMG, EGFR	NA	Initial CNS Tumor	NA	DMG, EGFR
-PT_FCWMT41S	7316-7133	BS_XK482X70	Diffuse astrocytic and oligodendroglial tumor	HGAT	NA	DMG, EGFR	NA	Initial CNS Tumor	NA	DMG, EGFR
+PT_FCWMT41S	7316-7133	BS_4Q7PQHS8	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-mutant	DMG, H3 K28	Updated via OpenPedCan subtyping	Initial CNS Tumor	NA	DMG, EGFR
```

Comment:
oh gosh - I thought we had updated this in v12, @zzgeng can you use "Diffuse midline glioma, H3 K28-altered" here, not "mutant" for all of these samples?